tests/apis/agency_api/v0_x/test_devices.py

```diff
@@ -170,6 +170,9 @@ def test_device_add(client):
     assert response.status_code == 201
     assert response.data == {}
     assert models.Device.objects.count() == 1
+    device = models.Device.objects.first()
```

Comment:
Nitpicking too, you can write a single line of `device = models.Device.objects.get()` that will fail if there is more than one object.